Yosemite/Yosemite/Model/Extensions/URL+Helpers.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  URL+Helpers.swift
+//  Yosemite
+//
+/// Convenient extension methods for URL
+import Foundation
+
+extension URL {
+    static public func getWooCommerceAdminURL(from storesManager: StoresManager) -> URL {
+        let wooCommerceFallbackURL = URL(string: "https://woocommerce.com/blog/")
```

Comment:
I wonder if this is the right place to add the extension, my rationale to put it in Yosemite is because we're retrieving this data directly from StoresManager, so made sense to add it here, but any feedback is appreciated! 

Additionally, I'm using the hardcoded string here as otherwise, I would need to import the WooCommerce framework just to access this constant, which seemed to be unnecessary.

Reply:
I think Yosemite might be a good place for this extension 👍 
What I am wondering is whether the extension should be on URL, or more specifically on `Site`. For me this method is strictly related to a Site, we are not using this anywhere else. 
On that, note the `StoresManager` as parameter is not very meaningful, if I see this function naming I wonder why a `StoresManager` instance, that basically manages the Flux Stores, has an admin URL.

You can take `Site+PluginsURL` as a reference, where we are doing something very similar. Actually, I would consider adding this new logic there, and collapsing these URL extensions into one, something like "Site+URL"

Reply:
I don't really understand why it's an extension on URL... did you consider an extension on StoresManager?

E.g. we don't use an extension on `String` to get the name of the store.

Reply:
Thanks to both for the input, that definitely makes much more sense than what I was doing. I'll move this extension under `Site+PluginsURL` and rename the file to reflect the update.